src/OpenLoco/src/Windows/PromptBrowseWindow.cpp

```diff
@@ -908,6 +922,9 @@ namespace OpenLoco::Ui::Windows::PromptBrowse
             case BrowseFileType::landscape:
                 _previewScenarioOptions = S5::readScenarioOptions(path);
                 break;
+            case BrowseFileType::heightmap:
+                //_previewScenarioOptions = S5::readScenarioOptions(path);
```

Comment:
unsure what to put here

Reply:
all I want from this "file browser" is the file path the user clicks - from there other code can load the png file